modules/sdk/gradle-plugins-ci-test-runner/build.gradle

```diff
@@ -0,0 +1,20 @@
+buildscript {
+
+	repositories {
+		mavenLocal()
+		maven {
+			url "https://repository-cdn.liferay.com/nexus/content/groups/public"
+		}
+	}
+}
+
+apply plugin: "com.liferay.base.plugin"
+
+String gradleVersion = "8.5"
+dependencies {
+	api group: "com.liferay", name: "com.liferay.ant.mirrors.get", version: "1.2.22"
+	api group: "com.liferay", name: "com.liferay.gradle.plugins", version: "16.1.61"
+	api group: "com.liferay", name: "com.liferay.gradle.plugins.defaults", version: "latest.release"
```

Comment:
Not sure you need to pull in the `defaults` plugin here - is that needed for something specific?